guides/doc-Configuring_Load_Balancer/topics/Registering_Clients.adoc

```diff
@@ -6,11 +6,17 @@ For more information about registering clients and configuring them to use Puppe
 
 To register clients, proceed to one of the following procedures:
 
+* xref:registering-clients-using-the-host-registration[]
 * xref:registering-clients-using-the-bootstrap-script[]
 * xref:registering-clients-manually[]
 
+[id='registering-clients-using-the-host-registration']
+== {ProjectName} host registration
+
+include::../common/modules/proc_host_registration_steps.adoc[leveloffset=+1]
+
 [id='registering-clients-using-the-bootstrap-script']
-== Registering Clients Using the Bootstrap Script
+== **(Deprecated)** Registering Clients Using the Bootstrap Script
```

Comment:
This also smells, @melcorr ?

Reply:
I've opened an issue for this: https://github.com/theforeman/foreman-documentation/issues/829